services/common/messages/telemetry.proto

```diff
@@ -24,6 +24,7 @@ message Overview {
     Velocity vel = 5;
     Speed speed = 6;
     Battery battery = 7;
+    string mode = 8; 
```

Comment:
Hmm, I wonder if we should enumerate the mode. This works for now though, and enumerating has complications because the custom mode differs based on the firmware.

Reply:
Can add an endpoint for returning the available modes for the autopilot in strings for the future?

Reply:
Also the file mode on this file changed in the commit